src/Drawer/BaseUtils.php

```diff
@@ -20,6 +20,10 @@ static function isAPrimitive($target) {
             is_null($target);
     }
 
+    static function isRemoval($value) {
```

Comment:
Can this live as a method in "handlecomponents" so that it doesn't bloat the utils? utils are kinda meant for more general utilities used in multiple places.

Reply:
Sure thing. I'll move it.